Batch.roboFontExt/lib/webFormats/__init__.py

```diff
@@ -251,7 +251,7 @@ def __init__(self, posSize, controller):
         y = 10
         for setting in self.webSettings:
             key = setting.replace(" ", "_").lower()
-            checkBox = CheckBox((10, y, -10, 22), setting,
+            checkBox = CheckBox((10, y, -10, 22), setting.replace("Save ", "").upper(),
```

Comment:
maybe the webSettings should just be list of supported formats, not titles.

Reply:
yeah, this was a quick UI change though, because the `save_` scheme was embedded in your structure, and I didn't want to mess anything up.

Reply:
Currently stable (IME), while refining the UI. maybe this is addressed when we do Batch 2.0?